### ISSUE TITLE
Style form using styled-components

### ISSUE THREAD

**Issue by thesanjeevsharma** [thesanjeevsharma/react-rrule#2]
Style the form for rrule using styled-components.

Features:
1.  Add a `color` prop to the component (add some default value if not passed).
2. Maintain a theme using that color using that component and style the complete form.

**Comment by pandurijal** [comment]
Can I take this issue? And just to make sure, the style would be totally the same, all you want to change is the way it coded (using styled component), right?

**Comment by thesanjeevsharma** [comment]
@pandurijal 
Nope, This isn't styled at all. It works functionally but styling is browser basic(no style at all).

It already includes styled-components. Just make it look pretty. 

As a user, I will call this component and pass color and it will blend in my react website. 

If you wanna take this, please confirm.

**Comment by thesanjeevsharma** [comment]
Nope, This isn't styled at all. It works functionally but styling is browser basic(no style at all).

It already includes styled-components. Just make it look pretty. 

As I user, I will call this component and pass color and it will blend in my react website. 

If you wanna take this, please confirm.
